crypto/evp_extra/p_kem_test.cc

```diff
@@ -831,3 +861,62 @@ TEST_P(KEMTest, ASN1_Methods_Cross_Compatibility) {
             Bytes(decoded_priv_from_marshal->pkey.kem_key->secret_key, test.secret_key_len));
 }
 
+TEST_P(KEMTest, ParsePrivateKeySeed) {
+  // Skip BouncyCastle entries and only test vectors from IETF standard
+  if (GetParam().public_pem_str == bouncy_castle_mlkem_512_pub_pem_str ||
+      GetParam().public_pem_str == bouncy_castle_mlkem_768_pub_pem_str) {
+    return;
+  }
```

Comment:
NP: It might be more robust to check specifically for `GetParam().public_pem_str == NULL || GetParam().private_pem_seed_str == NULL`.